schedule/qam/common/mau-extratests1.yaml

```diff
@@ -41,10 +41,12 @@ conditional_schedule:
   version_specific:
     VERSION:
       15-SP5:
+        - console/openssl_nodejs
```

Comment:
When it runs everywhere it can get out of version specific group :smile:

Reply:
Had the same idea, but we also have SLE 12 and it was never planned to run there